docs/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to4.0.6.md

```diff
@@ -4,25 +4,29 @@ description: Learn how you can migrate your Strapi application from 4.0.5 to 4.0
 canonicalUrl: https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to4.0.6.html
 ---
 
-# Migration guide from 4.0.5 to 4.0.6
+# v4.0.x to v4.0.6 migration guide
 
-**Make sure your server is not running until the end of the migration**
+The Strapi v4.0.x to v4.0.6 migration guide upgrades all prior versions of v4.0.x to v4.0.6. The migration adds the `session` middleware to the middleware array and configures the `session` middleware. The upgrade is required for the [Users & Permissions providers](/user-docs/latest/settings/configuring-users-permissions-plugin-settings.md) to function properly, secure cookies, and encrypt data. The migration guide consists of 3 sections: 
+  - upgrading the application dependencies
+  - migrating the breaking changes to the middleware
+  - reinitializing the application
 
-:::warning
-If you are using **extensions** to create custom code or modifying existing code, you will need to update your code and compare your version to the new changes on the repository.
-<br>
-Not updating your **extensions** can break your app in unexpected ways that we cannot predict.
+
+:::caution
+ [Plugins extension](/developer-docs/latest/plugins/users-permissions.md) that create custom code or modify existing code, will need to be updated and compared to the changes in the repository. Not updating the plugin extensions could break the app.
```

Comment:
```suggestion
 [Plugins extension](/developer-docs/latest/plugins/users-permissions.md) that create custom code or modify existing code, will need to be updated and compared to the changes in the repository. Not updating the plugin extensions could break the application.
```

You can just apply my suggestion, Shaun (I'll show you how to create this kind of PR suggestions if you want 🤗 )